setup.py

```diff
@@ -11,18 +11,17 @@
     description='A python implementation of OpenNMT',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    version='2.3.0',
+    version='3.0.0alpha',
     packages=find_packages(),
     project_urls={
         "Documentation": "http://opennmt.net/OpenNMT-py/",
         "Forum": "http://forum.opennmt.net/",
         "Gitter": "https://gitter.im/OpenNMT/OpenNMT-py",
         "Source": "https://github.com/OpenNMT/OpenNMT-py/"
     },
-    python_requires=">=3.5",
+    python_requires=">=3.7",
     install_requires=[
-        "torch>=1.9.0",
-        "torchtext==0.5.0",
+        "torch>=1.12.1",
```

Comment:
TBH I think it's best to set the last version because at times they remove some deprecated features and I prefer to be sure it's working fine with the last version. But yes we can test with two previous versions also.